s3/replication/manager/tests/system/load_transfer_test.py

```diff
@@ -40,18 +40,29 @@
 from s3replicationcommon.templates import fdmi_record_template
 from s3replicationmanager.config import Config as ManagerConfig
 from s3_config import S3Config
+import yaml
```

Comment:
move this above to after uuid, as a practise keep all the std imports before current package imports

Reply:
done